source/appendices/a-glossary-of-terms.rst

```diff
@@ -595,7 +599,11 @@ Note: The main Code of Practice document takes precedence in the case of any con
      -
 
    * - ISBN (International Standard Book Number)
-     - A unique standard identifier (ISO 2108) used to identify monographic publications (books).
+     - A unique standard identifier (ISO 2108) used to identify monographic publications (books). This would typically be in the format ISBN-13, with hyphens. E-ISBN is the expected value, with print ISBNs provided only where E-ISBN is not available.
```

Comment:
This seems to contradict the rule that the format ISBN-13 with hyphens is required for COUNTER reports.

Reply:
We do still see some old-style ISBN-10 in reports, hence 'typically'

Reply:
That is not permitted and will be flagged as an error by the Validation Tool.

Reply:
Error or critical error?

Reply:
Error, since fixing this should be easy - there are tools for many programming languages for converting and formatting ISBNs, including adding the correct hyphens based on the rules provided by the International ISBN Agency.